plugin/src/make-source-from-operation.ts

```diff
@@ -94,6 +94,26 @@ export function makeSourceFromOperation(
         objects.push(JSON.parse(line));
       }
 
+      if (pluginOptions.debugMode) {
+        const nodeTypeCounts = objects.reduce((grouping, object) => {
+          const [, remoteType] = object.id.match(idPattern);
+          if (!grouping[remoteType]) {
+            grouping[remoteType] = 0;
+          }
+
+          grouping[remoteType]++;
+          return grouping;
+        }, {});
+
+        const outpout = Object.entries(nodeTypeCounts)
```

Comment:
Probably meant to name this variable `output`? Thought outpout does sound cooler :-)